lib/utils/check-root-user.js

```diff
@@ -2,20 +2,53 @@
 
 const os = require('os');
 const chalk = require('chalk');
+const fs = require('fs');
+const includes = require('lodash/includes');
 
-function checkRootUser() {
-    // Skip if we're on windows
-    if (os.platform() !== 'linux') {
+const isRootInstall = function isRootInstall() {
+    const path = require('path');
+
+    if (!fs.existsSync(path.join(process.cwd(), '.ghost-cli'))) {
+        // CASE: .ghost-cli file doens't exist in the current wording directory
+        return false;
+    }
+
+    if (fs.statSync(path.join(process.cwd(), '.ghost-cli')).uid === 0) {
+        // .ghost-cli file exists and is owned by root
+        return true;
+    }
+
+    return false;
+}
+
+function checkRootUser(command) {
+    const allowedCommands = ['stop', 'start', 'restart'];
+    const isOneClickInstall = fs.existsSync('/root/.digitalocean_password');
+
+    if (os.platform() !== 'linux' || process.getuid() !== 0) {
         return;
     }
 
-    if (process.getuid() === 0) {
+    if (isOneClickInstall) {
+        // We have a Digitalocean one click installation
+        console.error(`${chalk.yellow('We discovered that you are using the Digitalocean One-Click install.')}
+    You need to create a user with regular account privileges and migrate your installation to use this user.
+    Please follow the steps here ${chalk.underline.green('https://docs.ghost.org/docs/troubleshooting#section-fix-root-user')} to fix your setup.`);
+    } else if (isRootInstall()) {
+        console.error(`${chalk.yellow('It seems installed Ghost as root user.')}
+You need to create a user with regular account privileges and migrate your installation to use this user.
+Please follow the steps here ${chalk.underline.green('https://docs.ghost.org/docs/troubleshooting#section-fix-root-user')} to fix your setup.`);
+    } else {
         console.error(`${chalk.yellow('Can\'t run command as \'root\' user.')}
-Please create a new user with regular account privileges and use this user to run the command.
-See ${chalk.underline.blue('https://docs.ghost.org/docs/install#section-create-a-new-user')} for more information`);
+Please use the user you set up in the installation process, or create a new user with regular account privileges and use this user to run 'ghost ${command}'.
+See ${chalk.underline.green('https://docs.ghost.org/docs/install#section-create-a-new-user')} for more information`);
+    }
 
+    if (!includes(allowedCommands, command)) {
```

Comment:
we talked about last night removing the "allowed commands" 3 or 4 versions from now - can you add a comment to this here saying as much?